docs/anomaly-detection/High Availability.md

```diff
@@ -0,0 +1,245 @@
+---
+weight: 3
+title: High Availability
```

Comment:
I'd suggest to rename this page to something like "Scaling vmanomaly" as it covers both high availability and sharding across multiple instances.